docs/administration/logging/standard-logging.md

```diff
@@ -0,0 +1,61 @@
+# Standard Logging
```

Comment:
Why is this file included?

Reply:
It shouldn't be. It's overlap with doc replatforming work. I'll get this out of the changeset before merging. Thanks for catching this.